sdk/core/azure-core-serializer-json-jackson/src/main/java/com/azure/core/serializer/json/jackson/JacksonJsonSerializerBuilder.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.serializer.json.jackson;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * Fluent builder class that configures and instantiates instances of {@link JacksonJsonSerializer}.
+ */
+public class JacksonJsonSerializerBuilder {
+    private ObjectMapper objectMapper;
+
+    /**
+     * Constructs a new instance of {@link JacksonJsonSerializer} with the configurations set in this builder.
+     *
+     * @return A new instance of {@link JacksonJsonSerializer}.
+     */
+    public JacksonJsonSerializer build() {
+        return (objectMapper == null)
+            ? new JacksonJsonSerializer(new ObjectMapper())
+            : new JacksonJsonSerializer(objectMapper);
+    }
+
+    /**
+     * Sets the {@link ObjectMapper} that will be used during serialization.
+     * <p>
+     * If this is set to {@code null} the default {@link ObjectMapper} will be used.
+     *
+     * @param objectMapper {@link ObjectMapper} that will be used during serialization.
+     * @return The updated JacksonJsonSerializerBuilder class.
+     */
+    public JacksonJsonSerializerBuilder objectMapper(ObjectMapper objectMapper) {
```

Comment:
This name might be more appropriate than `gson` is.

Reply:
I would agree, the name here is better.